CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-<a name="0.1.0"></a>
+# Changelog
```

Comment:
@mn-42 We should either update the changelog or get rid of it altogether.

Reply:
changed it... good?